### DISCUSSION TITLE
Player Notes without GM

### DISCUSSION THREAD

**Discussion by CiverBlack** [vigoren/foundryvtt-simple-calendar#257] [Q&A]
Hi,
my players write Sessionsummarys but can not add them to the calendar when I am not present. Is there a setting or could there be one to allow players to add notes when I am offline? Would really help me thanks.

**Comment by vigoren** [comment]
Hey right now because of how notes are stored the GM had to be logged in (the gm account actually does the saving)

That said I'm currently working on a large update that will be moving the notes to be sure as journal entries which means players should be able to add notes without needing the GM present. So when the next update is released this should be resolved.
